shared/js/background/classes/ad-click-attribution-policy.js

```diff
@@ -109,31 +120,38 @@ export class AdClick {
      * @param {number} navigationExpiration in seconds
      * @param {number} totalExpiration in seconds
      */
-    constructor (navigationExpiration, totalExpiration) {
+    constructor (navigationExpiration, totalExpiration, tabId, allowlist) {
+        console.log("Create Ad Click")
+
         /** @type {string | null} */
         this.adBaseDomain = null
         this.adClickRedirect = false
         this.navigationExpiration = navigationExpiration
         this.totalExpiration = totalExpiration
-        this.expires = Date.now() + (this.totalExpiration * 1000)
+        //this.expires = Date.now() + (this.totalExpiration * 1000)
+        this.expires = Date.now() + (1000 * 1000)
         this.clickExpires = Date.now() + (this.navigationExpiration * 1000)
+
+        this.adClickDNR = null
     }
 
     clone () {
-        const adClick = new AdClick(this.navigationExpiration, this.totalExpiration)
+        const adClick = new AdClick(this.navigationExpiration, this.totalExpiration, this.tabId)
         adClick.adBaseDomain = this.adBaseDomain
         adClick.adClickRedirect = this.adClickRedirect
         adClick.expires = this.expires
         adClick.clickExpires = Date.now() + (this.navigationExpiration * 1000)
+        adClick.adClickDNR = this.adClickDNR
```

Comment:
I think when we clone, we probably want to remove the rule and re-create perhaps?